datalad/__init__.py

```diff
@@ -22,6 +22,11 @@
     import random
     random.seed(_seed)
 
+from .config import ConfigManager
+cfg = ConfigManager()
+
+from .log import lgr
+
```

Comment:
I think there might be a side effect of colorama not overloading sys.stdout then if lgr imported before.

In general, order of imports in `__init__.py` is "fragile" so I would advice to not mess with it unless really needed, and otherwise delay until actual used.  e.g. here you could leave cfg where it was, and add additional from .log import lgr right before your lgr.warning before .